src/CoreBundle/Controller/UserController.php

```diff
@@ -8,21 +8,48 @@
 
 use Chamilo\CoreBundle\Repository\Node\IllustrationRepository;
 use Chamilo\CoreBundle\Repository\Node\UserRepository;
+use CourseManager;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Attribute\Route;
 use Symfony\Component\Security\Core\User\UserInterface;
+use UserGroupModel;
 
 /**
  * @author Julio Montoya <gugli100@gmail.com>
  */
-#[Route('/user')]
```

Comment:
Removing `/user` is not a super nice idea. Better leave it and use `/user/overview` as path for `overview()`.
Paths with "main" are *only* kept until all the legacy code is migrated to Vue